sklearn/datasets/tests/test_samples_generator.py

```diff
@@ -113,7 +113,7 @@ def test_make_classification_informative_features():
         (2, [1 / 2] * 2, 2),
         (2, [3 / 4, 1 / 4], 2),
         (10, [1 / 3] * 3, 10),
-        (int(64), [1], 1),
+        (64, [1], 1),
```

Comment:
Can anyone think of a reason why it would make sense to explicitly write `int(64)` instead of `64`? The previous lines all use `10` or `2` directly, this makes me wonder if there is some subtle effect that needs `int(64)`. At least it seems like the author of the code knew about these two ways of specifying an integer.